xml/gl.xml

```diff
@@ -5006,36 +5006,36 @@ typedef unsigned int GLhandleARB;
     </enums>
 
     <enums namespace="GL" group="FragmentShaderDestMaskATI" type="bitmask">
-        <enum value="0x00000001" name="GL_RED_BIT_ATI"/>
-        <enum value="0x00000002" name="GL_GREEN_BIT_ATI"/>
-        <enum value="0x00000004" name="GL_BLUE_BIT_ATI"/>
+        <enum value="0x00000001" name="GL_RED_BIT_ATI" group="FragmentShaderDestMaskATI"/>
```

Comment:
Inheriting the group attribute from the surrounding enums tag won't work?

Reply:
Currently it's not defined in the RNC spec, and all other bitfields have group attributes anyway. While it could be inherited, it would require more spec changes and a lot of tweaks in dependent code to adapt to the change.